person.rb

```diff
@@ -1,18 +1,22 @@
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
 class Nameable
   def correct_name
     raise NotImplementedError, 'Subclasses must implement this method'
   end
 end
 
 class Person < Nameable
-  attr_accessor :id, :name, :age
+  attr_reader :rentals
 
   def initialize(age:, name: 'unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] A person has many rentals. kindly ensure a method for adding rentals should be present. Here the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```